gcp_variant_transforms/transforms/variant_to_bigquery.py

```diff
@@ -41,39 +43,38 @@ def process(self, record):
 class VariantToBigQuery(beam.PTransform):
   """Writes PCollection of ``Variant`` records to BigQuery."""
 
-  def __init__(self, output_table, header_fields, variant_merger=None,
-               split_alternate_allele_info_fields=True, append=False,
-               omit_empty_sample_calls=False,
-               annotation_fields=None):
+  def __init__(
+      self,
+      output_table,  # type: str
+      header_fields,  # type: vcf_header_parser.HeaderFields
+      variant_merger=None,  # type: variant_merge_strategy.VariantMergeStrategy
+      proc_var_factory=None,  # type: processed_variant.ProcessedVariantFactory
+      append=False,  # type: bool
+      omit_empty_sample_calls=False  # type: bool
+  ):
     """Initializes the transform.
 
     Args:
-      output_table (str): Full path of the output BigQuery table.
-      header_fields (``HeaderFields``): A ``namedtuple`` containing
-        representative header fields for all ``Variant`` records.
-        This is needed for dynamically generating the schema.
-      variant_merger (``VariantMergeStrategy``): The strategy used for merging
-        variants (if any). Some strategies may change the schema, which is why
-        this may be needed here.
-      split_alternate_allele_info_fields (bool): If true, all INFO fields with
-        `Number=A` (i.e. one value for each alternate allele) will be stored
-        under the `alternate_bases` record. If false, they will be stored with
-        the rest of the INFO fields.
-      append (bool): If true, existing records in output_table will not be
+      output_table: Full path of the output BigQuery table.
+      header_fields: A `namedtuple` containing representative header fields for
+        all variants. This is needed for dynamically generating the schema.
+      variant_merger: The strategy used for merging variants (if any). Some
+        strategies may change the schema, which is why this may be needed here.
+      proc_var_factory: The factory class that knows how to convert Variant
+        instances to ProcessedVariant. As a side effect it also knows how to
+        modify BigQuery schema based on the ProcessedVariants that it generates.
+        The latter functionality is what is needed here.
+      append: If true, existing records in output_table will not be
         overwritten. New records will be appended to those that already exist.
       omit_empty_sample_calls (bool): If true, samples that don't have a given
```

Comment:
nit: can you also remove bool from here? :)

Reply:
Done.